kivy/uix/actionbar.py

```diff
@@ -14,8 +14,9 @@
 
 An :class:`ActionBar` contains an :class:`ActionView` with various
```

Comment:
We should change also ``` An :class:`ActionBar` contains an :class:`ActionView` ``` as suggested by the issue author?

Reply:
Thank you. I have fixed this now.